lib/gollum-lib/filter/macro.rb

```diff
@@ -47,7 +48,9 @@ def process(data)
         begin
           Gollum::Macro.instance(macro, @markup.wiki, @markup.page).render(*args)
         rescue StandardError => e
-          "!!!Macro Error: #{e.message}!!!"
+          icon = Octicons::Octicon.new('zap', {width: 24, height: 24})
+          icon.options[:class] << ' mr-2'
+          "<div class='flash flash-error'>#{icon.to_svg}!!!Macro Error for #{macro}: #{e.message}!!!</div>"
```

Comment:
Now that there's a nice shiny red alert box, couldn't we get rid of the triple exclamation marks on both sides?

Reply:
Yup, will do.